lib/resources/kernel_module.rb

```diff
@@ -21,10 +21,11 @@ def initialize(modulename = nil)
     end
 
     def loaded?
-      # default lsmod command
-      lsmod_cmd = 'lsmod'
-      # special care for CentOS 5 and sudo
-      lsmod_cmd = '/sbin/lsmod' if inspec.os[:name] == 'centos' && inspec.os[:release].to_i == 5
+      if inspec.os[:family] == 'redhat'
```

Comment:
We should try to avoid using `inspec.os[:family] `. A better approach is to use `inspec.os.redhat?`